src/ansys/fluent/core/workflow.py

```diff
@@ -75,19 +75,14 @@ def ordered_children(self) -> list:
         """Get the ordered task list held by this task. Sorting is in terms
         of the workflow order and only includes this task's top-level tasks, while other tasks
         can be obtained by calling ordered_children() on a parent task. Given the
-        workflow:
-
-        o Workflow
-        |
-        |--o A
-        |
-        |--o B
-        |  |
-        |  |--o C
-        |  |
-        |  |--o D
-        |
-        |--o E
+        workflow::
+
+            Workflow
+            ├── A
+            ├── B
+            │  ├── C
+            │  └── D
+            └── E
```

Comment:
This resolves the warnings `:7: (WARNING/2) Inline substitution_reference start-string without end-string.` during doc generation. Output:

![image](https://user-images.githubusercontent.com/94432368/226784068-07021a13-6380-489b-8680-a4a99023a166.png)